README.md

```diff
@@ -7,67 +7,88 @@
 
 **WebAssembly Smart Contracts for the Cosmos SDK**
 
-This repo provides a useful functionality to build smart contracts that
-are compatible with a Cosmos SDK based runtime, [`wasmd`](https://github.com/CosmWasm/wasmd).
+This repo provides a useful functionality to build smart contracts that are
+compatible with a Cosmos SDK based runtime,
+[wasmd](https://github.com/CosmWasm/wasmd).
 
 Compatibility:
 
-* Contracts built with CosmWasm `v0.6.3+` will run on `wasmd` `v0.6.x`
-* Contracts built with CosmWasm `v0.7.x` will run on `wasmd` `v0.7.x`
+- Contracts built with CosmWasm `v0.6.3+` will run on `wasmd` `v0.6.x`
+- Contracts built with CosmWasm `v0.7.x` will run on `wasmd` `v0.7.x`
```

Comment:
Please add a comment that master is pre-0.8 and incompatible with all `wasmd` releases (we change this when 0.8 is out)

Reply:
Done in 44ab7d8